### ISSUE TITLE
Help with simple command aliases

### ISSUE THREAD

**Issue by Xiione** [PYRRH4/CustomCommands#7]
### Issue description
The commands file is very difficult to understand. None of the provided example commands in the file demonstrate how an "execute commands as" command should be formatted.
What I am trying to do is override minecraft's default "/stats" command, and replace it for BattleLevels's "/level" command. I am trying to make it so that "/stats" will accept one optional argument, as "/level" does the same by default.

Basically, I want to make "/stats [player]" do the same thing as "/level [player]".
### Versions

Spigot : git-Spigot-4bd94dc-c765646 (MC: 1.12.2)

PyrCore : 3.2.3

CustomCommands : 2.3.2

### Log file

None needed, as this is merely a request for clarification regarding some of the plugin's features.

**Comment by Nesseley** [comment]
https://pastebin.com/6HQg3cGs
Here is how I did it for making /statz player go to /votes player.
Should be fine just editing it to your choise! I had the same problem as you so I agree the examples doesnt help at all

**Comment by Xiione** [comment]
Thanks, it worked!
Pyr, you might want to take some notes! XD

**Comment by PYRRH4** [comment]
Ikr, I'm sorry for the lack of response ^^ that guy was way faster ^^
